.github/workflows/ci_test-conda.yml

```
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.8"]  # previous to last Python version as that one is already used in test-full
-        pytorch-version: ["1.6", "1.7", "1.8", "1.9", "1.10"]
+        pytorch-version: ["1.7.1", "1.8", "1.9", "1.10"]
```

```suggestion
        pytorch-version: ["1.7", "1.8", "1.9", "1.10"]
```
That should work since we test with 3.8 here, no?